Formula/terraform.rb

```diff
@@ -31,6 +31,10 @@ class Terraform < Formula
     url "https://go.googlesource.com/tools.git", :revision => "977844c7af2aa555048a19d28e9fe6c392e7b8e9"
   end
 
+  # This patch works around a known test failure, which will be fixed in terraform version 0.6.17.
+  # https://github.com/hashicorp/terraform/issues/6709
+  patch :DATA
```

Comment:
Just link to https://github.com/hashicorp/terraform/commit/6a20e8 as a Ruby comment and we'll ship this.